aten/src/THC/generic/THCTensorMasked.cu

```diff
@@ -165,6 +247,67 @@ void THCTensor_(maskedSelect)(THCState* state,
   THCudaCheck(cudaGetLastError());
 }
 
+void THCTensor_(maskedSelectBool)(THCState* state,
```

Comment:
also for the future: I think you could move these definitions into a non-generic function so you don't have to copy the code.